tests/admin-ui-tests/utils.ts

```diff
@@ -76,7 +76,11 @@ export const deleteAllData: (projectDir: string) => Promise<void> = async (proje
 
     let prisma = new PrismaClient();
 
-    await Promise.all(Object.values(prisma).map((x: any) => x?.deleteMany?.({})));
+    await prisma.$transaction(
+      Object.values(prisma)
+        .filter((x: any) => x?.deleteMany)
+        .map((x: any) => x?.deleteMany?.({}))
+    );
```

Comment:
Doing things in transactions also gets around the SQLite write timeout immediately thing so I've done that here